### DISCUSSION TITLE
User Interface

### DISCUSSION THREAD

**Discussion by Zeppelin-PV** [solectrus/solectrus#1874] [Ideas]
Suggestions for improvement and implementation requests at Solectrus are currently piling up.
From this I see a great need for an operator interface.
Many of the most recently received inquiries could be solved with it.

 -   custom color scheme
 -   Import of historical data
 -   Push notification in case of temperature problems
 -   Parameters of the PV system
 -   Geo data for forecast
 -   Display kW-->Watt
 -   Euro in cents (there often 3-4 digits after the decimal point)
 -  place two graphics on top of each other to display relevant processes
     e.g. Storage in % and charging current
- save status "battery diagnostic" with day and duration to get an overall list on this matter
- time range on daily power production only from sunrise till sunset, night hours should not displayed
- selectable time range on "Jetzt/Now" one, two and three hours
- show production daily peak value - maybe in top ten
- show costs/feed-in payment in TopTen in electricity purchase/PV generation

Maybe there are other interesting points for a UI.

Many thanks to Georg for the excellent tool
good description of the installation, even for newcomers to Raspi & Co

**Comment by GalOR7** [comment]
Hallo Georg, würde nochmals die Idee mit der Visualisierung weiterer Strings MPP2 / MPP3 anbringen. Ich weiss du hast dazu nicht viel Zeit und keinen Vorteil für dich weil du in deiner Anlage eben einen Sting hast. Aber vielleicht rückt die Idee ja auf der "Wishlist" nach oben :-)

**Reply by ledermann** [reply]
Das Thema wird in #1756 behandelt, hat offen gestanden derzeitig keine hohe Priorität. Derzeitig fokussiere ich mich auf die Anbindung separater Verbraucher (#2239). Danach mal sehen, es gibt viele interessante Themen ;-)
